### PR TITLE
quczer/manifold

### LINKED ISSUE #7: create manifold

### PR DESCRIPTION
**Pull request by quczer**

### CODE REVIEW

File: src/latent_geometry/path.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+from functools import cache
+from typing import Callable, Final, Optional
+
+import numpy as np
+
+
+class Path:
```
Comment (marmarmarmar) [inline src/latent_geometry/path.py]: SolverResultPath

File: src/latent_geometry/solver/exponential.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        dy / dt = (v, a) = (v, a(x, v))
+        ```
+        so
+        ```
+        f(t, y) = f(t, (x, v)) = (v, acceleration_fun(x, v))
+        ```
+        """
+        t_span = (0.0, 1.0)
+        y0 = self._pack_state(position, velocity)
+        fun = self._create_fun(acceleration_fun)
```
Comment (marmarmarmar) [inline src/latent_geometry/solver/exponential.py]: _create_solver_fun ?

File: src/latent_geometry/path.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            xs.append(self(t))
+            vs.append(self.velocity(t))
+            accs.append(self.acceleration(t))
+        return xs, vs, accs
+
+    @property
+    def length(self) -> float:
+        return Path._integrate_length(self.velocity)
+
+    @staticmethod
```
Comment (marmarmarmar) [inline src/latent_geometry/path.py:79]: Why is this static? It makes sense for it to be standard object method.

File: src/latent_geometry/solver/logarithm.py [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        ```
+
+        Notes
+        -----
+        We will evaluate the solution in a couple of predefined points ti, therefore
+        y is always of shape (2*D, k), where D is the dimension of our space,
+        k - some number of mesh nodes the solver desires to evaluate in.
+        """
+        t_span = np.linspace(0.0, 1.0, self.n_mesh_nodes)
+        y = self._create_y(start_position, finish_position)
```
Comment (marmarmarmar) [inline src/latent_geometry/solver/logarithm.py]: I would name y in a more meaningful manner.

File: src/latent_geometry/path.py (hunk 3 above)
Comment (quczer) [reply src/latent_geometry/path.py:79]: Dont we want to have static methods wherever we can? This way we communicate that the method will not change the state of the object and its easier to test.

**Pull request merged**